targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/sleep.c

```diff
@@ -28,8 +28,11 @@ void hal_sleep(void)
 void hal_deepsleep(void)
 {
 #if (defined(FSL_FEATURE_SOC_MCG_COUNT) && FSL_FEATURE_SOC_MCG_COUNT)
+#if defined(kMCG_ModePEE)
```

Comment:
If this is one commit  (might be fine for this type of change) - please include as much details as you can . I would recommend to include what warnings are fixed here. For instance, what this ModePEE does ? how is it related to CLOCK_GetMode() ?